File: InBoxOutBoxExample/Core/InBoxOutBoxExample.Domain/Write/Entities/Product.cs
namespace InBoxOutBoxExample.Domain.Write.Entities
{
    public class Product : BaseEntity
    {
        public string Name { get; set; }
        public string Brand { get; set; }
        public string Category { get; set; }
        public string? Description { get; set; }
        public int Price { get; set; }

    }
}

File: consumer-service/internal/model/product.go
package model

import "time"

type Product struct {
	Id          int       `gorm:"primarykey" bson:"id"`
	CreatedDate time.Time `bson:"createddate"`
	UpdatedDate time.Time `bson:"updateddate"`
	Name        string    `bson:"name"`
	Brand       string    `bson:"brand"`
	Category    string    `bson:"category"`
	Description string    `bson:"description"`
	Price       int       `bson:"price"`
}
